### ISSUE TITLE
ATOM assumes the name of the calibration package when running the calibration

### ISSUE THREAD

**Issue by Kazadhum** [lardemua/atom#653]
When running the calibration, for example, for the `zau2` system:

`rosrun atom_calibration calibrate -json $ATOM_DATASETS/zau2/e5/dataset1/dataset.json -v`

We get the following error:

```
Traceback (most recent call last):
  File "/home/diogo/catkin_ws/devel/lib/atom_calibration/calibrate", line 15, in <module>
    exec(compile(fh.read(), python_script, 'exec'), context)
  File "/home/diogo/catkin_ws/src/atom/atom_calibration/scripts/calibrate", line 497, in <module>
    main()
  File "/home/diogo/catkin_ws/src/atom/atom_calibration/scripts/calibrate", line 493, in main
    saveResultsXacro(dataset, selected_collection_key, list(transforms_set))
  File "/home/diogo/catkin_ws/src/atom/atom_core/src/atom_core/xacro_io.py", line 82, in saveResultsXacro
    path_to_file = (rospack.get_path(dataset["_metadata"]["robot_name"] + "_calibration") + "/urdf/optimized/")
  File "/usr/lib/python3/dist-packages/rospkg/rospack.py", line 207, in get_path
    raise ResourceNotFound(name, ros_paths=self._ros_paths)
rospkg.common.ResourceNotFound: zau2_calibration
ROS path [0]=/opt/ros/noetic/share/ros
ROS path [1]=/home/diogo/catkin_ws/src
ROS path [2]=/opt/ros/noetic/share
INFO - 2023-10-19 17:40:31,893 - core - signal_shutdown [atexit]
```

It seems like ATOM is guessing the name of the calibration package apparently based on the name of the robotic system in the URDF files (?). It doesn't seem to be based on the calibration `config.yml` file, at the least.

Tagging @miguelriemoliveira for visibility.

**Comment by manuelgitgomes** [comment]
Hello! 

So I think a good solution for this is to, as I said in #680, add the package name in configuration,
To make it compatible with old datasets, I can add a clause that if the package name is not on the dataset, use the old option.

@miguelriemoliveira volunteered to do this, but I think I can do this now quickly.

What do you think?

**Comment by manuelgitgomes** [comment]
Hello! 
Since this a new change to datasets, I created a new branch for this, named `issue653`.
The changes are detailed in #682.

I ask one of you at least (@Kazadhum and @miguelriemoliveira) to review these changes and merge them to our main branch.

**Comment by Kazadhum** [comment]
Hello @manuelgitgomes! After I get off from classes I can take a look at the changes and test them as well. From the description of your pull request it seems like it will not raise any issues. I'll try to give some feedback on it this evening.

**Comment by manuelgitgomes** [comment]
Closed with #682
